### DISCUSSION TITLE
Implementing into angular application

### DISCUSSION THREAD

**Discussion by cgraves09** [magicbell/community#61] [Questions] [answered]
Hi ive been running into an issue and cant seem to get around using the @magicbell/embeddable npm

The error i keep getting is 
TS7016: Could not find a declaration file for module '@magicbell/embeddable'. '/workspace/ui-v2/node_modules/@magicbell/embeddable/dist/magicbell.min.js' implicitly has an 'any' type. Try `npm i --save-dev @types/magicbell__embeddable` if it exists or add a new declaration (.d.ts) file containing `declare module '@magicbell/embeddable';` 7 import { renderWidget } from '@magicbell/embeddable'; 

Ive added the .d.ts file and declared it but still wont compile

**Comment by unamashana** [comment]
Hey @cgraves09 I am sending this over to some folks who could help. Thanks for your patience.

**Comment by smeijer** [comment, marked as answer]
Hi @cgraves09,

Unfortunately, we don't publish types for that package at this moment. Please disable type checking for the embeddable till we have this sorted out.
